rotkehlchen/tests/unit/decoders/test_cowswap.py

```diff
@@ -50,7 +50,7 @@ def test_swap_token_to_token(ethereum_inquirer, ethereum_accounts):
             asset=A_WBTC,
             balance=Balance(amount=FVal(raw_amount)),
             location_label=user_address,
-            notes=f'Swap {raw_amount} WBTC in cowswap',
+            notes=f'Swap {raw_amount} WBTC in cowswap market order',
```

Comment:
Can you change the decoder to fix the wording here? Shouldn't it have an `a` here?
```suggestion
            notes=f'Swap {raw_amount} WBTC in a cowswap market order',
```